x-pack/plugins/security_solution/public/cases/components/timeline_actions/add_to_case_action.test.tsx

```diff
@@ -218,7 +222,38 @@ describe('AddToCaseAction', () => {
       alertId: 'test-id',
       index: 'test-index',
       rule: {
-        id: null,
+        id: 'rule-id',
+        name: 'rule-name',
+      },
+      type: 'alert',
+    });
+  });
+
+  it('it set rule information as null when missing', async () => {
+    const wrapper = mount(
+      <TestProviders>
+        <AddToCaseAction
+          {...props}
+          ecsRowData={{
+            _id: 'test-id',
+            _index: 'test-index',
+            signal: { rule: { id: ['rule-id'], false_positives: [] } },
+          }}
+        />
+      </TestProviders>
+    );
+
+    wrapper.find(`[data-test-subj="attach-alert-to-case-button"]`).first().simulate('click');
+    wrapper.find(`[data-test-subj="add-new-case-item"]`).first().simulate('click');
```

Comment:
nit: You don't need to interpolate here, you can just use `'[data-test-subj="add-new-case-item"]'`